include/ts/apidefs.h.in

```diff
@@ -597,34 +597,6 @@ typedef enum {
   AFTER_BODY,
 } TSFetchWakeUpOptions;
 
-#ifndef _HTTP_PROXY_API_ENUMS_H_
-#define _HTTP_PROXY_API_ENUMS_H_
-/// Server session sharing values - match
-/// Must be identical to definition in HttpProxyAPIEnums.h
-typedef enum {
-  TS_SERVER_SESSION_SHARING_MATCH_NONE,
-  TS_SERVER_SESSION_SHARING_MATCH_BOTH,
-  TS_SERVER_SESSION_SHARING_MATCH_IP,
-  TS_SERVER_SESSION_SHARING_MATCH_HOST
-} TSServerSessionSharingMatchType;
-
-/// Server session sharing values - pool
-/// Must be identical to definition in HttpProxyAPIEnums.h
-typedef enum {
-  TS_SERVER_SESSION_SHARING_POOL_GLOBAL,
-  TS_SERVER_SESSION_SHARING_POOL_THREAD,
-} TSServerSessionSharingPoolType;
-
-/// Values for per server outbound connection tracking group definition.
-/// See proxy.config.http.per_server.match
-typedef enum {
-  TS_SERVER_OUTBOUND_MATCH_IP,
-  TS_SERVER_OUTBOUND_MATCH_PORT,
-  TS_SERVER_OUTBOUND_MATCH_HOST,
-  TS_SERVER_OUTBOUND_MATCH_BOTH
-} TSOutboundConnectionMatchType;
-#endif
```

Comment:
So plugins don't need any of these to do overrides?

Reply:
Good point.  TSServerSessionSharingMatchType is the the value that would be used in records.config or for conf remap.  The setting with TSServerSessionSharingPoolType is not overridable.

I'll put back TSOutboundConnectionMatchType